packages/docmanager-extension/src/index.ts

```diff
@@ -336,13 +336,64 @@ export const pathStatusPlugin: JupyterFrontEndPlugin<void> = {
   }
 };
 
+/**
+ * A plugin providing download commands in the file menu and command palette.
+ */
+export const downloadPlugin: JupyterFrontEndPlugin<void> = {
+  id: '@jupyterlab/docmanager-extension:download',
+  autoStart: true,
+  requires: [ITranslator, ICommandPalette, IMainMenu, IDocumentManager],
```

Comment:
Should `ICommandPalette` and `IMainMenu` be moved to `optional`?

Reply:
Yup--sorry, missed that, just moved it!